capstone-ui/src/Components/App/App.jsx

```diff
@@ -41,12 +51,18 @@ export default function App() {
                   ))}
               </div>
               <h2 className="title">Website Name</h2>
-              <div className='user-info'>
+              <div className='login-section'>
                 {user ? (
-                  <>
-                  <span>Welcome, {user.username}! |</span>
-                  <button onClick={handleLogout}>Logout</button>
-                  </>
+                  <div className='profile'>
+                    <div className='user-info'>
+                      <div>{user.username}</div>
+                      <div>Zip Code: {user.zipcode}</div>
+                      <div className="following-list" onClick={handleFollowingListClick}>Candidates Followed: {user.following.length}</div>
+                    </div>
+                    <div className='logout'>
+                      <button onClick={handleLogout}>Logout</button>
+                    </div>
+                  </div>
```

Comment:
Would this component only exist on the login screen?

Reply:
This component exists in the home screen after a user is logged in.